app/main/routes.py

```diff
@@ -130,10 +130,10 @@ def browse():
         sorting_orders = build_sorting_orders()
 
     if transferring_body_id:
+        print("i was here")
         browse_type = "transferring_body"
         browse_parameters["transferring_body_id"] = transferring_body_id
-        # filters["series"] = "tsta1"
-        # filters["date_range"] = {"date_from": "01/08/2022", "date_to": "31/08/2022"}
+        filters = _build_browse_all_filters()
```

Comment:
Should we rename `_build_browse_all_filters` since it is used here as well and will be used in browse series too?

Reply:
this has been renamed to moved to separate file in PR 177

Reply:
yeah  just saw that now 👍